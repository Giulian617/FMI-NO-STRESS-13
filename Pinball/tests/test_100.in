100 0
24 1 10
1 66 2
28 1 1
95 1 6
1 76 8
72 1 2
7 1 5
6 1 1
10 1 3
1 9 6
60 1 9
1 26 5
1 80 10
1 73 1
1 83 2
81 1 8
49 1 3
29 1 3
1 71 7
54 1 7
1 75 1
14 1 8
1 51 6
78 1 5
42 1 8
1 88 2
3 1 4
68 1 10
27 1 2
87 1 1
12 1 2
38 1 7
1 79 3
1 31 9
1 59 10
1 98 6
1 32 1
1 52 1
1 33 2
1 4 4
30 1 1
44 1 6
1 18 3
55 1 4
1 22 4
1 46 3
1 2 6
1 17 10
1 58 9
1 47 1
84 1 6
69 1 7
1 91 9
8 1 6
1 39 3
1 94 5
1 65 1
67 1 6
23 1 3
1 100 2
62 1 7
56 1 10
89 1 3
70 1 1
1 96 5
19 1 10
1 40 1
1 90 9
1 77 10
1 50 2
57 1 1
34 1 7
15 1 10
1 53 8
1 92 5
61 1 1
13 1 6
1 35 10
20 1 4
1 74 6
1 63 10
48 1 9
21 1 1
1 45 8
11 1 10
5 1 9
43 1 6
86 1 2
93 1 4
1 16 9
85 1 10
1 36 3
37 1 10
25 1 3
97 1 8
1 82 10
41 1 4
64 1 3
1 99 6
