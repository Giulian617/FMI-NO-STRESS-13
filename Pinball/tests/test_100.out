810246685
206831121
206831121
715370024
301707782
508538903
810246685
810246685
905123346
17077799
603415564
810246685
413662242
17077799
715370024
17077799
905123346
17077799
206831121
301707782
206831121
905123346
17077799
905123346
508538903
603415564
301707782
905123346
301707782
715370024
301707782
603415564
111954460
17077799
905123346
17077799
111954460
905123346
301707782
206831121
413662242
810246685
810246685
413662242
905123346
301707782
715370024
905123346
603415564
810246685
301707782
413662242
111954460
206831121
17077799
301707782
715370024
17077799
715370024
301707782
111954460
17077799
905123346
301707782
603415564
810246685
17077799
111954460
301707782
111954460
603415564
301707782
810246685
301707782
413662242
17077799
508538903
905123346
17077799
413662242
17077799
603415564
810246685
17077799
603415564
301707782
603415564
905123346
715370024
715370024
508538903
206831121
508538903
810246685
508538903
413662242
810246685
810246685
603415564
